test/zdtm/static/cpu-affinity0.desc

```diff
@@ -0,0 +1 @@
+{'dopts': '', 'ropts': '--with-cpu-affinity', 'flags': 'reqrst '}
```

Comment:
If dump opts are empty probably we don't need them at all? And one more thing - Why do you need reqrst flag? What happens if you remove it and run the test with --norst?